### PR TITLE
Add setup script with explicit paths to installation locations

### PR DESCRIPTION
**Pull request by mindyls**
Include a simple script that is intended to be used as reference / modified by individual users, based on their specific installation locations.  This script does not expect any relationships between installation directories (ie, conda installation path, test data repos, source repos, cartopy shapefiles, etc), but is intended as a guide for users to configure their own environment.

```
$ source $HOME/geoips/geoips/setup/USER_MODIFY_config
(geoips) $ which python
$HOME/mambaforge/envs/geoips/bin/python
(geoips) $ $GEOIPS_PACKAGES_DIR/geoips/tests/integration_tests/full_test.sh
```

### CODE REVIEW

File: docs/source/releases/v1_11_3a0.rst
Comment (mindyls) [inline docs/source/releases/v1_11_3a0.rst:18]: NOTE: Do not merge until other PRs are merged, and this one is automatically updated to point to main!

File: setup/USER_MODIFY_config
Comment (cpcamacho) [inline setup/USER_MODIFY_config:8]: I guess "eval" is needed here to work? Probably not ideal, but likely fine?

**Review by cpcamacho — APPROVED** [review]

File: setup/USER_MODIFY_config
Comment (jsolbrig) [reply setup/USER_MODIFY_config:8]: I wonder if that line should be commented out by default?

File: docs/source/releases/v1_11_3a0.rst
Comment (jsolbrig) [reply docs/source/releases/v1_11_3a0.rst:18]: Done and ready to go.

File: setup/USER_MODIFY_config
Comment (mindyls) [reply setup/USER_MODIFY_config:8]: It is the line that conda tells you to run if you tell it not to initialize the shell

Comment (mindyls) [reply setup/USER_MODIFY_config:8]: Yeah,  sure,  comment it out by default

Comment (jsolbrig) [reply setup/USER_MODIFY_config:8]: I think I'm going to approve this as a "hidden Mindy-only feature" for now. She uses this for her environment initialization. I've got my own methods, too.

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**
